src/main/java/tc/oc/pgm/Config.java

```diff
@@ -274,4 +280,65 @@ public static ChatColor finished() {
       }
     }
   }
+
+  public static class Flairs implements Listener {
+    private boolean configMode;
+    private Map<String, Flair> flairs = new HashMap<String, Flair>();
+
+    @EventHandler
+    public void onConfigLoad(ConfigLoadEvent event) throws InvalidConfigurationException {
+      this.configMode = event.getConfig().getBoolean("flair.config-mode", true);
```

Comment:
`config-mode` -> `enabled`

Edit: See my idea for `groups`, this would no longer be needed. If no `flair` is defined, flairs aren't enabled.

Reply:
Could you expand on this?

Reply:
`groups` is just the syntax for `Config`, keep everything else flair related for your code. No mention of groups.

Reply:
So if there are no groups defined in the config, flairs are completely disabled? Or flairs are only used from the API?

Reply:
Yep. No groups, no flairs. No flair for a group, that group has no flairs.